modules/jwplayerBidAdapter.js

```diff
@@ -50,4 +94,113 @@ export const spec = {
   // onBidderError: function({ error, bidderRequest }) {}
 };
 
+function buildRequest(bidRequest, bidderRequest) {
+  // Open RTB Request Object
```

Comment:
nit: redundant comment. It's best to only leave comments when they add value and clarity. Usually comments are great for explaining business logic since it might not be reflected in the code. 
In this case however, i think it's clear that we are building an ortb request object because the var name is perfect. So I would say we can do without the comment :)

Reply:
removed.